forge-gui/res/cardsfolder/rebalanced/a-ready_to_rumble.txt

```diff
@@ -0,0 +1,8 @@
+Name:A-Ready to Rumble
+ManaCost:4 R
+Types:Instant
+A:SP$ Charm | Choices$ DBDmg,DBDestroy
+SVar:DBDmg:DB$ DealDamage | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | NumDmg$ 5 | StaticAbilities$ STCantPrevent | NoPrevention$ True | SpellDescription$ Damage can't be prevented this turn. CARDNAME deals 5 damage to target creature or planeswalker.
```

Comment:
DealDamage won't handle `StaticAbilities`... I'd make an Effect with the StaticAbilities, then SubAbility to DealDamage

The `NoPrevention` isn't needed as the card points to an overall "damage can't be prevented" (handled by the effect) not a specific "damage can't be prevented"

Reply:
this is stomp, probably important: `AILogic$ Burn`
```
A:SP$ Effect | Cost$ 1 R | Name$ Stomp Effect | StaticAbilities$ STCantPrevent | AILogic$ Burn | SubAbility$ DBDamage | SpellDescription$ Damage can't be prevented this turn. CARDNAME deals 2 damage to any target.
SVar:STCantPrevent:Mode$ CantPreventDamage | EffectZone$ Command | Description$ Damage can't be prevented.
SVar:DBDamage:DB$ DealDamage | ValidTgts$ Player,Planeswalker,Creature | TgtPrompt$ Select any target | NumDmg$ 2 | NoPrevention$ True
```

hm for this one, `NoPrevention` also shouldn't be needed 🤔